### DISCUSSION TITLE
DepthCor showing the first 2 components with high correlation with sequencing depth

### DISCUSSION THREAD

**Discussion by rauldiul** [stuart-lab/signac#1302] [Q&A] [answered]
Hi! Thanks for your software

I'm using `DepthCor` on some merged scATAC data (4 merged libraries), and I see that the 2nd component is still quite correlated with depth (>0.5 cor):

![image](https://user-images.githubusercontent.com/51365483/204806799-b0e7f4e0-9aef-46bf-9723-90659bac0f2e.png)

(I also see this behavior on the separate datasets so it is not driven by batch, this is also apparent when coloring the clusters by batch)

Are you familiar with obtaining such high correlation values for the 2nd component? Are there instances where you have filtered both the first and the second components?

This is what the clustering looks by sequence depth, removing the 1st component:

![image](https://user-images.githubusercontent.com/51365483/204809108-bc96c3d2-8338-4053-8793-6946edb256ec.png)

And this is removing the 1st + 2nd component, I don't think there's much difference:

![image](https://user-images.githubusercontent.com/51365483/204809235-19673a16-1909-4435-828e-e616dd46ca50.png)

thanks a lot for your help

**Comment by timoast** [comment, marked as answer]
Personally I would only remove the component with the highest correlation with sequencing depth. The correlation between depth and the second LSI component is much lower than that of the first component, and LSI2 might be capturing some important biological variance in the dataset.

**Reply by rauldiul** [reply]
Thanks for the feedback! I agree.
